Source/SwiftLintFramework/Rules/FirebaseConfigFetchRule.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  FirebaseConfigFetchRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigFetchRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_fetch",
+        name: "Firebase Config Fetch",
+        description: "Firebase Config fetch should be called in viewDidLoad().",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+            "        (status, error) -> Void in \n" +
+            "    }\n" +
+            "  }\n" +
+            "}"
+        ],
+        triggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "  }\n" +
+            "  func fetch {\n " +
+            "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+            "        (status, error) -> Void in \n" +
+            "    }\n" +
+            "  }\n" +
+            "}"
+        ]
+    )
+
+    public func validate(file: File, kind: SwiftExpressionKind,
+                         dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        guard SwiftExpressionKind.call == kind else {
+            return []
+        }
+
+        guard dictionary.name!.hasSuffix(".fetch") else {
+            return []
+        }
+
+        guard dictionary.substructure[0].name == "withExpirationDuration" else {
```

Comment:
It's the case, they ignored the the other rule of putting fetch(withExperiration) in viewDidload.
So if there is no viewDidLoad at all.
Added an example.